File: 202101/main.py


def solution():
    ip =  open("input.txt", "r").readlines()
    lastSeq = int(ip[0]) + int(ip[1]) + int(ip[2])
    counter = 0

    for i in range(1, 1998):
      currSeq = int(ip[i]) + int(ip[i + 1]) + int(ip[i + 2])

      if currSeq > lastSeq:
        counter += 1

      lastSeq = currSeq
     #print("current sequence: ", currSeq)

    print(counter)

if __name__ == '__main__':
    solution()

File: 202101/main.cs
using System;

public class solution
{
    static void Main(string[] args){
	string[] lines = System.IO.File.ReadAllLines("input.txt");
	Console.WriteLine(Solution1(lines));
}  
    static int Solution1(string[] lines){
	int counter = 0;
	int last  = int.Parse(lines[0]);
	foreach(string line in lines){
	   int a = int.Parse(line);
	   if(a > last)
		counter++;
	   last = a;		
	}
	return counter;
}
}
